server/controller/cycleData/index.ts

```diff
@@ -1,7 +1,9 @@
 import { persistNodeValue } from './persistNodeValue'
 import { getTableData } from './getTableData'
+import { getOriginalDataPointData } from './getOriginalDataPointData'
 
 export const CycleDataController = {
   persistNodeValue,
   getTableData,
+  getOriginalDataPointData,
```

Comment:
this can just be CycleDataRepository.getOriginalDataPointData and remove ./getOriginalDataPointData